src/components/CreateNewProjectWizard/SummaryPane.js

```diff
@@ -31,9 +30,7 @@ class SummaryPane extends PureComponent<Props> {
       return (
         <IntroWrapper>
           <FadeIn key="intro-t">
-            <Swimming>
-              <Logo size="large" />
-            </Swimming>
+            <Logo grayscale size="large" />
```

Comment:
Curious why the grayscale here instead of the regular colored version?

Reply:
It matches with the Next button, and there's something to be said for the similarities between the lack of color in the logo and the lack of content in a new project.

That being said, personally I think the colored version fits best here, but I can definitely see both sides.

Reply:
My thinking was just that the contrast between the yellow-orange logo and the blue-green gradient background was a bit harsh:

<img width="1118" alt="screen shot 2018-08-23 at 7 26 48 am" src="https://user-images.githubusercontent.com/6692932/44522877-ee24bd80-a6a5-11e8-9d14-fad8170999fa.png">

Looking at it now, though, it doesn't seem too bad? What do you think, @melanieseltzer and @superhawk610?

Also @superhawk610, I wonder if you've tried other colour palettes for the logo? The old logo was yellow-turquoise-aqua (on MacOS), and it meant it blended in a bit more in situations like this. I actually really like the colours you've chosen, but I wonder if it might also look good in those colours?

Reply:
I personally love the color version there 🙂 I think it looks great!